src/main.c

```diff
@@ -253,6 +253,10 @@ static struct {
     size_t num_listeners;
     char *pid_file;
     char *error_log;
+    struct {
+        int listening_fd;
```

Comment:
```suggestion
        int listen_fd;
```
I'm concerned this sticks out, I do not think we use ~ing elsewhere.